packages/core/test/state.test.ts

```diff
@@ -331,14 +331,14 @@ describe('State', () => {
     });
   });
 
-  describe('State.create()', () => {
+  describe('machine.createState()', () => {
     it('should be able to create a state from a JSON config', () => {
       const { initialState } = exampleMachine;
       const jsonInitialState = JSON.parse(JSON.stringify(initialState));
 
-      const stateFromConfig = State.create(jsonInitialState) as StateFrom<
-        typeof exampleMachine
-      >;
+      const stateFromConfig = exampleMachine.createState(
+        jsonInitialState
+      ) as StateFrom<typeof exampleMachine>;
```

Comment:
Those casts are no longer relevant after the `State.create`->`machine.createState` change